snyk/cli
# feat: restructure the issues list layout

#### What does this PR do?
We’d like to restructure the layout of the issues list.
The changes include:
1. Shortening the current title, “Infrastructure as code issues:”, to “Issues“ (without colon)
2. Instead of having a list of issues per file, we would like to have a list of issues per severity level.
These sections should be sorted from highest severity to lowest.
Each section should have a title with the number of the issues detected with the corresponding severity. Example: “Medium Severity Issues: 2”.
The titles should be colorized based on the severity they represent. The color mapping can be found in [this design](https://miro.com/app/board/uXjVOOpVGxU=/?moveToWidget=3458764518430874390&cot=14).

#### How should this be manually tested?
Test an IaC configuration files without the FF and check that the output didn't change.
Turn on the FF, test the IaC files once again and check that the new output gets printed.

#### What are the relevant tickets?
https://snyksec.atlassian.net/jira/software/c/projects/CFG/boards/301?modal=detail&selectedIssue=CFG-1577

#### Screenshots
![image](https://user-images.githubusercontent.com/71096571/162607971-be85a0ff-4881-4b5a-9fcd-c5a7d14066c6.png)
